src/lib/schedule.c

```diff
@@ -102,3 +102,12 @@ int schedule_sof_init(void)
 
 	return ret;
 }
+
+void schedule_free(void)
+{
+	/* free edf scheduler */
+	edf_scheduler_free();
+
+	/* free system llq */
+	free_system_llq();
+}
```

Comment:
Same here free_system_llq -> system_llq_free();

Reply:
It has been renamed to ll_scheduler_free().